src/SFA.DAS.EmployerAccounts.Api.Client/EmployerAccountsApiClient.cs

```diff
@@ -1,12 +1,13 @@
-﻿using System.Collections.Generic;
-using System.Threading;
+﻿using System.Threading;
 using System.Threading.Tasks;
+using Newtonsoft.Json;
+using SFA.DAS.EmployerAccounts.Api.Types;
 using SFA.DAS.EmployerAccounts.ReadStore.Application.Queries;
 using SFA.DAS.EmployerAccounts.ReadStore.Mediator;
-using SFA.DAS.EmployerAccounts.Types.Models;
 
 namespace SFA.DAS.EmployerAccounts.Api.Client
 {
+    //todo: not a great client/http combo
```

Comment:
I agree and it needs fixing before it gets worse.  I'll add a separate task as it'll be a pretty chunky job.

Reply:
Sorry, meant to comment yesterday...

We agreed a while ago within the team not to use the client package and just do a http request instead as the client offers little than a maintenance overhead.  We're fine using the types package tho.

Reply:
I've removed the usage of the client and introduced a service to talk to the employer accounts API instead.